wezterm-input-types/src/lib.rs

```diff
@@ -1976,6 +1976,7 @@ bitflags! {
         const MACOS_FORCE_DISABLE_SHADOW = 4;
         const MACOS_FORCE_ENABLE_SHADOW = 4|8;
         const INTEGRATED_BUTTONS = 16;
+        const MACOS_RESPECT_THEME_BACKGROUND = 32;
```

Comment:
naming wise, we don't use the term `theme` to refer to color schemes or color preferences, so this is an inconsistency.

I'd prefer it for the name to spell out what it does, which to me sounds like `MACOS_USE_BACKGROUND_COLOR_AS_TITLEBAR_COLOR` or something like that, as that is what this PR appears to be doing

Reply:
done